### PR TITLE
Replace old `master` branch with `main`

### PR DESCRIPTION
**Pull request by mathroule**
## Description

Replace `master` branch with `main` branch in code.

## Related Issues

- https://github.com/markdownlint/markdownlint/pull/465

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation (non-breaking change that does not add functionality but updates documentation)
- [x] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [x] I have read the [**CONTRIBUTING**](https://github.com/markdownlint/markdownlint/blob/master/CONTRIBUTING.md) document.
- [x] Wrote [good commit messages](https://chris.beams.io/posts/git-commit/)
- [x] Feature branch is up-to-date with `master`, if not - rebase it
- [x] Added tests for all new/changed functionality, including tests for positive and negative scenarios
- [x] The PR relates to *only* one subject with a clear title and description in grammatically correct, complete sentences

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -98,9 +98,9 @@ respond to your pull request with any feedback they might have. The process at
 this point is as follows:
 
 1. A review is required from at least one of the project maintainers. See the
-   master maintainers document for Markdownlint project at
```
Comment (mathroule) [inline CONTRIBUTING.md:101]: Should it be `main` instead?

**Review by jaymzh — APPROVED** [review]

**Comment by jaymzh** [comment]
Thanks! You need to rebase though.

**Comment by mathroule** [comment]
RuboCop reported offenses which I has to fix to get a green checks on CI: https://github.com/markdownlint/markdownlint/actions/runs/6107184659/job/16573692044#step:5:8.

**Comment by mathroule** [comment]
> Thanks! You need to rebase though.

Done

**Review by jaymzh — APPROVED** [review]

**Pull request merged**